README.md

```diff
@@ -9,11 +9,253 @@ GT.M modifications - Sam Habiel.
+
+Separate operators with commas and/or spaces. There may be an optional trailing comma.
+
+An implicit and group surrounds the entire filter query parameter.
+
+###### Grouping Operators
+
+Grouping operators can be nested. For example, or(and(eq("foo","bar"),like("baz","quux")),in("ping","pong"))
+
+An implicit and group surrounds the entire filter query parameter. For example, the following 2 filter query parameters are equivalent:
+
+filter=eq(make,"Lincoln"),eq(model,"Town Car")
+filter=and(eq(make,"Lincoln"),eq(model,"Town Car"))
+
+Grouping operators can have any number of operands.
+
+|Grouping operators|
+|------------------|
+|and(operators)    |
+|or(operators)     |
+|not(operators)    |
+
+###### Comparison Operators
+
+Comparison operators take arguments inside parentheses. The first argument is always what JDS field to inspect and the second argument is what value to check for.
+Arguments can't be empty. Lists are comma-separated items that start with [ and end with ]
+
+NOTE: all JDS filters must be submitted to JDS URI-encoded. The following examples are not URI-encoded.
+
+|Operator |Arguments                |Usage                                      |Example                              |
+|---------|-------------------------|-------------------------------------------|-------------------------------------|
+|eq       |(field,value)            |equals, exact comparison                   |eq(make,"Lincoln")                   |
+|ne       |(field,value)            |not equals, exact comparison               |ne(make,"Lincoln")                   |
+|in       |(field,[list,of,values]) |inside list, exact comparison with list    |in(make,["Lincoln","Buick"])         |
+|nin      |(field,[list,of,values]) |not inside list, exact comparison with list|nin(make,["Hyundai","Toyota"])       |
+|exists   |(field)                  |exists                                     |exists(make)                         |
+|gt       |(field,value)            |greater than                               |gt(year,1978)                        |
+|gte      |(field,value)            |greater than or equal                      |gte(year,1978)                       |
+|lt       |(field,value)            |lesser than                                |lt(year,1978)                        |
+|lte      |(field,value)            |lesser than or equal                       |lte(year,1978)                       |
+|between  |(field,low,high)         |between exclusive                          |between(year,1978,1999)              |
+|like     |(field,value)            |M pattern match, case sensitive            |like(model,"Continental%")           |
+|ilike    |(field,value)            |M pattern match, case insensitive          |ilike(model,"CONTINENTAL%")          |
+
+% is the wildcard character
+
+# Documentation To-Do List
+
+In the future, this documentation will also cover indexes and templates, as time permits.
```

Comment:
I'll have to add more information about templates. There is a ton of power in templates as well (can customize what data is returned/formatted).

Reply:
If time permits, I may dig in and try to figure out how to use them in the context of GDS, but I'm not there yet. The initial use case in my application (where I would add it is performing splendidly) is content with basic CRUD operations + filters, so I haven't gotten there yet.

I think once I get into building my reporting engine, I'll have a greater motivation to dig.